File: src/main/java/FlyAway/client/dto/ClientReservationDto.java
package FlyAway.client.dto;

import FlyAway.reservation.dto.ReservationWithoutUserDto;

import java.time.LocalDate;
import java.util.List;

public record ClientReservationDto(
        String firstname,
        String lastname,
        String email,
        String phoneNumber,
        LocalDate dayOfBirth,
        List<ReservationWithoutUserDto> reservations
) {
}

File: frontend/src/app/services/models/client-reservation-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ReservationWithoutUserDto } from '../models/reservation-without-user-dto';
export interface ClientReservationDto {
  dayOfBirth?: string;
  email?: string;
  firstname?: string;
  lastname?: string;
  phoneNumber?: string;
  reservations?: Array<ReservationWithoutUserDto>;
}
